Questions.txt

```diff
@@ -407,3 +407,10 @@ Q36: Write a code to print daimond pattern in any langauge.
   
 System.out.println(" I am commiting for first time")
 
+Q37: What is a database ?
+
+Q38 : What’s the difference between HTML, HTML5, CSS and CSS3 ?
+
+Q39 : What is the difference between front-end and back-end development ?
+
+Q40 : What is an API ?
```

Comment:
Hi @MoniSwon! Thanks for you PR!
For information, as the opposite of French, there is no space before `!` and `?` punctuation marks in English 😉.

Reply:
Hi Mathieu!
Is my push in the Pull Request ? I'm not sure I've done it well but it seems it worked

Reply:
Yep! It's ok now! You also could have amend your first commit and force push it.